src/compile.c

```diff
@@ -518,7 +519,7 @@ dcc_build_somewhere(char *argv[],
 
     /* FIXME: this may leak memory for argv. */
 
-    ret = dcc_scan_args(argv, &input_fname, &output_fname, &new_argv);
+    ret = dcc_scan_args(argv, &input_fname, &output_fname, &new_argv,&hasgcov);
```

Comment:
Please add a space after the comma in lists.

Reply:
This lines has been resolved.